### DISCUSSION TITLE
Filter page content based on user context

### DISCUSSION THREAD

**Discussion by leyan** [docsifyjs/docsify#2386] [Q&A]
Is it possible to filter displayed content based on some attribute sent in the URL? I am generating a product documentation and I would like some users to see everything, while other would not see advanced sections or sections requiring permissions they do not have. These sections can be represented with dedicated markup in the source Markdown files.

**Comment by paulhibbitts** [comment]
Hi @leyan , not that I know but this recent Docsify Plugin might provide a few new options to explore: https://github.com/FlippedCodes/docsify-betterembed
